app/pb_kits/playbook/pb_popover/_popover.jsx

```diff
@@ -156,13 +161,15 @@ export default class PbReactPopover extends React.Component<PbPopoverProps> {
             )}
           </PopperReference>
         </If>
-        <If condition={usePortal}>
-          {ReactDOM.createPortal(
-            popoverComponent,
-            document.querySelector(portal)
-          )}
-          <Else />
-          {popoverComponent}
+        <If condition={show}>
```

Comment:
The only other alternative here is not a good one at all and is potentially the reason for the memory consumption. That is using `forceUpdate` which will cause render recursion.